redesign_tests/profiles.py

```diff
@@ -0,0 +1,13 @@
+'''
+Contains instantiated structured and unstructured profiles
+that will be used as inputs in unit tests.
+'''
+
+import dataprofiler as dp
+import pandas as pd
+
+tab_data = dp.Data("redesign_tests/tabular.csv")
+tab_profile = dp.Profiler(tab_data)
+
+unstruct_data = pd.Series(['first string', 'second string'])
+unstruct_profile = dp.Profiler(unstruct_data, profiler_type='unstructured')
```

Comment:
I think it might be better to be in the tests as `setUpClass`
```python
@classmethod
def setUpClass(cls):
    .
    .    
    .
```

Reply:
- agree on the `setUpClass`. we can talk about this and show example in `DataProfiler`
- also, new line EOF